target/linux/ramips/image/mt7621.mk

```diff
@@ -2094,6 +2094,15 @@ define Device/netis_wf2881
 endef
 TARGET_DEVICES += netis_wf2881
 
+define Device/openfi_5pro
+  $(Device/dsa-migration)
+  IMAGE_SIZE := 63448k
```

Comment:
Where does this size come from? The size of firmware partition in the device tree is `0x3fb0000` (66,781,184 bytes = 65,216 KiB).